subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/report/listener/LogcatBuffer.kt

```diff
@@ -49,6 +53,14 @@ internal interface LogcatBuffer {
             tailer.stop()
         }
 
+        override fun getStdout(): List<String> {
```

Comment:
Why `getLogs` was not enough for clients? 
It return an interface which is better when raw collection.

Reply:
i don't know; maybe refactor it later
somehow i couldn't do coroutines wrapping with interface